posthog/hogql/property.py

```diff
@@ -72,7 +72,7 @@ def visit_call(self, node: ast.Call):
 def property_to_expr(
     property: Union[BaseModel, PropertyGroup, Property, dict, list, ast.Expr],
     team: Team,
-    scope: Literal["event", "person", "session", "replay"] = "event",
+    scope: Literal["event", "person", "session", "replay", "replay_entity"] = "event",
```

Comment:
Works for us for now but I wonder if we need to talk to @PostHog/team-product-analytics about how they intend this to be used. Conscious we've added two new scopes during this work